pkg/auth/webapp/ui_param.go

```diff
@@ -1,19 +1,21 @@
 package webapp
 
 import (
+	"errors"
 	"net/http"
 
+	"github.com/authgear/authgear-server/pkg/lib/oauth/oauthsession"
 	"github.com/authgear/authgear-server/pkg/lib/uiparam"
 	"github.com/authgear/authgear-server/pkg/util/httputil"
 	"github.com/authgear/authgear-server/pkg/util/intl"
 )
 
-// ClientIDCookieDef is a HTTP session cookie.
-var ClientIDCookieDef = &httputil.CookieDef{
-	NameSuffix: "client_id",
-	Path:       "/",
-	SameSite:   http.SameSiteNoneMode,
-}
+// ClientIDCookieDef is deprecated.
```

Comment:
Why don't we simply remove it? 🤔

Reply:
Leaving the lines here to prevent us from re-adding it in the future. Also for easier `git blame`